content/applications/finance/accounting/bank/feeds/enablebanking.rst

```diff
@@ -0,0 +1,41 @@
+===============================================
+Enable Banking as bank synchronization provider
+===============================================
+
+**Enable Banking** is a third party provider aggregating banking information from bank accounts all
+in one place.
+
+.. image:: enablebanking/enablebanking.png
+   :align: center
+   :alt:   Enable Banking logo
+
+**Enable Banking** offers non-intrusive connectivity to ASPSPs’ official APIs across Europe without
+storing data. Odoo synchronizes directly with banks to get access to all bank transactions and
+automatically import them into your database.
+
+Configuration
+=============
+
+Link bank accounts with Odoo
+----------------------------
+
+In Odoo, all bank transactions are directly synchronized and automatically imported into your
+database.
```

Comment:
This is redundant with what you wrote just before, in the introduction.

Either delete it or write something relevant to this section.